object_types: ['door', 'traffic_cone']
object_file_locations: ['/files/doors_full_map.csv', '/files/traffic_cones.csv']
object_publisher_topic: ['map_doors', 'detected_cones']
